app/src/main/java/com/hackslash/haaziri/models/Host.java

```diff
@@ -1,4 +1,39 @@
 package com.hackslash.haaziri.models;
 
 public class Host {
+    public String name;
+    public String email;
+    public String phone;
+    public String uld;
+
+    public Host() {
+    }
+
+    public Host(String a,String b,String c){
```

Comment:
what have you done here... your have not initialized the member variables of the class... you should do something like name = a;
email = b;
phone = c;
also try to use releavent naming for the varible, don't use a, b, c because when you would be creating an object of the class, you might be confused that in which order were you needed to pass the parameters...